fabric-server-languages-v1/src/main/java/net/fabricmc/fabric/impl/server/languages/v1/ServerLanguagesImpl.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.server.languages.v1;
+
+import net.fabricmc.api.DedicatedServerModInitializer;
+
+public class ServerLanguagesImpl implements DedicatedServerModInitializer {
```

Comment:
Don't think this will be necessary for this mod.

Reply:
Also removed